akka-persistence/src/main/resources/reference.conf

```diff
@@ -105,40 +105,47 @@ akka.persistence {
 
       # Dispatcher for the plugin actor.
       plugin-dispatcher = "akka.persistence.dispatchers.default-plugin-dispatcher"
-      
+
       # Dispatcher for message replay.
       replay-dispatcher = "akka.persistence.dispatchers.default-replay-dispatcher"
-      
+
       # Removed: used to be the Maximum size of a persistent message batch written to the journal.
       # Now this setting is without function, PersistentActor will write as many messages
       # as it has accumulated since the last write.
       max-message-batch-size = 200
-      
+
+      # There is more time in between individual events gotten from the journal
+      # during replay than this the replay will be failed and behave which causes
```

Comment:
Yeah, I left it inbetween two ways of phrasing it and forgot to complete. I'll rephrase.